app/portainer/settings/authentication/components/SaveAuthSettingsButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { LoadingButton } from '@/portainer/components/Button/LoadingButton';
+
+export interface Props {
+  onSaveSettings(): void;
```

Comment:
since this is `SaveAuthSettingsButton` the props names don't require the full name, as it's obvious from the component name. so `onSubmit`, `state`, `disabled`

Reply:
fixed